torch_xla/distributed/zero_redundancy_optimizer.py

```diff
@@ -91,17 +95,36 @@ def init_zero(self):
           group = list(group)
         self.local_rank = group.index(self.global_rank)
     if self.local_rank is None:
-      raise ValueError(
-          f"Current rank {self.global_rank} is missing from the sharding_groups {self.sharding_groups}"
-      )
+      raise ValueError(f"Current rank {self.global_rank} is missing from the sharding_groups {self.sharding_groups}")
     # Shard parameters for use in optimizer
     sharded_param_groups = self._shard_parameters()
     # Optimizer initialization
+    # Here we pop the differentiable default because the adam family of
+    # optimizers don't have differentiable as an argument. This should
+    # be fixed by this commit https://github.com/pytorch/pytorch/pull/86183
+    # and should be available in torch==2.0. For 1.13, we are patching it here.
```

Comment:
Hi @amithrm , here it says "This should be fixed by this commit https://github.com/pytorch/pytorch/pull/86183 and should be available in torch==2.0." Can you remove this patch?

Reply:
yes

Reply:
@jeffhataws PTAL